cmd/pgo-rmdata/process.go

```diff
@@ -389,9 +513,22 @@ func removeAddons(request Request) {
 	pgbouncerDepName := request.ClusterName + "-pgbouncer"
 
 	deletePropagation := metav1.DeletePropagationForeground
-	_ = request.Clientset.
-		AppsV1().Deployments(request.Namespace).
-		Delete(ctx, pgbouncerDepName, metav1.DeleteOptions{PropagationPolicy: &deletePropagation})
+	if err := request.Clientset.AppsV1().Deployments(request.Namespace).
+		Delete(ctx, pgbouncerDepName,
+			metav1.DeleteOptions{PropagationPolicy: &deletePropagation}); err != nil {
+		log.Error(err)
+		return
+	}
+
+	if err := wait.Poll(time.Second, time.Minute, func() (bool, error) {
+		if _, err := request.Clientset.AppsV1().Deployments(request.Namespace).
+			Get(ctx, pgbouncerDepName, metav1.GetOptions{}); err == nil || !kerror.IsNotFound(err) {
+			return false, nil
+		}
+		return true, nil
+	}); err != nil {
+		log.Error("could not terminate the pgBackRest repo deployment")
```

Comment:
```suggestion
		log.Error("could not terminate the pgBouncer repo deployment")
```

Reply:
Changed "pgBackRest repo" to "pgBouncer".